### ISSUE TITLE
[Suggestion]: Falling Blocks in the Bowels

### ISSUE THREAD

**Issue by SimonGamingPl** [nonamecrackers2/crackers-wither-storm-mod#1395]
### Desription of feature request

It's just a new command block attack. Sometimes a pile of blocks may fall from the roof trying to crush the player

I think its ez to make and isnt complicated

this my 5ft idea and i want to die

**Comment by TaintedBlockmObile** [comment]
This would make the fight really dangerous. I kinda like it. Should def be a config tho

**Comment by Pandaboi25x** [comment]
reminds me of that one ws scratch project cool idea tho

**Comment by BluePineappleGuy** [comment]
it could also be really good for ambience, random blocks falling from the ceiling when the storm shakes

**Comment by MFNmodding** [comment]
or when the command block gets damaged, blocks fall from the ceiling.

**Comment by kaboom24** [comment]
Yes but only if it is a config option other than that I really like it
